### PR TITLE
feat(bundler): add Sepolia bundler

### PR DESCRIPTION
**Pull request by Rubilmax**
Sepolia bundler

### CODE REVIEW

**Review by Jean-Grimal — APPROVED** [review]

File: src/sepolia/SepoliaBundler.sol [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity 0.8.21;
+
+import {GoerliLib} from "./libraries/GoerliLib.sol";
```
Comment (MerlinEgalite) [inline src/sepolia/SepoliaBundler.sol:4]:
```suggestion
import {SepoliaLib} from "./libraries/SepoliaLib.sol";
```

File: src/sepolia/SepoliaBundler.sol [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    Permit2Bundler,
+    ERC4626Bundler,
+    WNativeBundler,
+    UrdBundler,
+    MorphoBundler,
+    ERC20WrapperBundler
+{
+    /* CONSTRUCTOR */
+
+    constructor(address morpho) WNativeBundler(GoerliLib.WETH) MorphoBundler(morpho) {}
```
Comment (MerlinEgalite) [inline src/sepolia/SepoliaBundler.sol:32]:
```suggestion
    constructor(address morpho) WNativeBundler(SepoliaLib.WETH) MorphoBundler(morpho) {}
```

File: src/sepolia/libraries/SepoliaLib.sol [hunk 3]
```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+library SepoliaLib {
+    /// @dev The address of the WETH contract on Ethereum.
+    address internal constant WETH = 0x7b79995e5f793A07Bc00c21412e50Ecae098E7f9;
```
Comment (MerlinEgalite) [inline src/sepolia/libraries/SepoliaLib.sol:6]: this address seems wrong, maybe this address:0x7b79995e5f793a07bc00c21412e50ecae098e7f9

**Review by MerlinEgalite — CHANGES_REQUESTED** [review]

**Review by MerlinEgalite — APPROVED** [review]

File: src/sepolia/libraries/SepoliaLib.sol [hunk 4]
```diff
@@ -4,4 +4,7 @@ pragma solidity ^0.8.0;
 library SepoliaLib {
     /// @dev The address of the WETH contract on Ethereum.
```
Comment (MerlinEgalite) [inline src/sepolia/libraries/SepoliaLib.sol:5]:
```suggestion
    /// @dev The address of the WETH contract on Sepolia.
```

File: src/sepolia/libraries/SepoliaLib.sol [hunk 5]
```diff
@@ -4,4 +4,7 @@ pragma solidity ^0.8.0;
...
     address internal constant WETH = 0x7b79995e5f793A07Bc00c21412e50Ecae098E7f9;
+
+    /// @dev The address of the wstETH contract on Ethereum.
```
Comment (MerlinEgalite) [inline src/sepolia/libraries/SepoliaLib.sol:8]:
```suggestion
    /// @dev The address of the wstETH contract on Sepolia.
```

**Review by MerlinEgalite — CHANGES_REQUESTED** [review]

**Review by MerlinEgalite — APPROVED** [review]

**Review by Jean-Grimal — APPROVED** [review]

File: src/sepolia/SepoliaBundler.sol [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+/// @notice Bundler contract specific to the Sepolia testnet.
+contract SepoliaBundler is
+    TransferBundler,
+    PermitBundler,
+    Permit2Bundler,
+    ERC4626Bundler,
+    WNativeBundler,
+    StEthBundler,
```
Comment (MathisGD) [inline src/sepolia/SepoliaBundler.sol:27]: ?

Comment (Rubilmax) [reply src/sepolia/SepoliaBundler.sol:27]: I deployed a mock since, so the comment of this PR was outdated!

**Review by julien-devatom — APPROVED** [review]

**Review by MathisGD — APPROVED** [review]

**Pull request merged**